### PR TITLE
Clarify distance and near behavior when outside the sensing range

### PR DESCRIPTION
**Pull request by anssiko**
PTAL @riju @Honry @kaixinjxq

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/proximity/pull/31.html" title="Last updated on Jan 17, 2018, 10:49 AM GMT (658ecb5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/proximity/31/10d7f05...658ecb5.html" title="Last updated on Jan 17, 2018, 10:49 AM GMT (658ecb5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -129,9 +132,11 @@ The {{ProximitySensor/near!!attribute}} attribute of the {{ProximitySensor}}
 interface returns the result of invoking [=get value from latest reading=] with
 <emu-val>this</emu-val> and "near" as arguments.
 
+If the physical object is outside the <a>sensing range</a>, the attribute must return null.
+
 Note: If the implementation is unable to provide the [=near=] value, it could infer the [=near=]
 value from the value of [=distance=]. For example, if [=distance=] is not equal to [=max=], it
-could imply there is an object in the sensing range.
+could imply there is a physical object in the sensing range.
```
Comment (Honry) [inline index.bs]: nit: make `sensing range`  within `<a>` tag.

Comment (anssiko) [reply index.bs]: Thanks! Fixed in https://github.com/w3c/proximity/pull/31/commits/658ecb504456ae63869f9e417c273d155ae6d54d

**Comment by anssiko** [comment]
@kaixinjxq, does this address your comments?

**Comment by xiuqijix** [comment]
Yes, thanks @anssiko.

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Comment by anssiko** [comment]
Thanks @kaixinjxq and @Honry!

**Pull request merged**